lib/_tools.scss

```diff
@@ -296,7 +292,10 @@
      * Disable the flexbox grid
      */
     .#{$gel-grid-namespace}layout--no-flex {
-        display: block;
+        &,
```

Comment:
Would it be possible to have this modifier class merged sooner? We've realised we need it for our Top Stories build.